### PR TITLE
Awesome-Books Refactoring using modules

### PR DESCRIPTION
**Pull request by sumon766**
- Refactor the whole code using modules
- Create different modules for each item
- Export and import items from different modules
- Kept all functionalities without errors

### CODE REVIEW

File: modules/navigation.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+export const navigation = () => {
+  const header = document.getElementById('menu');
+  const btns = header.getElementsByClassName('btn');
+
+  for (let i = 0; i < btns.length; i += 1) {
+    btns[i].addEventListener('click', function () {
```
Comment (TedLivist) [inline modules/navigation.js]: - [ ] Kindly use ES6 syntax (arrow) for your functions.

File: modules/allBook.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/prefer-default-export */
+export class AllBook {
```
Comment (TedLivist) [inline modules/allBook.js]: - [ ] Please avoid disabling linters. Linters are important for maintaining coding standards across projects. You fix this error like so:

  - Remove the linter disabling.
  - Change line 2 to this:
    
    ```
    export default class AllBook {
    ```
  - Then in the import in `src/index.js` file, you should change to this:
  
    ```
    import AllBook from './modules/allBook.js';
    ```

File: modules/navigation.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (TedLivist) [inline modules/navigation.js]: - [ ] Kindly remove this linter disabling code and solve the error manually.

File: modules/time.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (TedLivist) [inline modules/time.js]: - [ ] Kindly remove this linter disabling code and solve the error manually.

File: index.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (TedLivist) [inline index.js]: - [ ] Kindly remove this linter disabling code and solve the error manually.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @sumon766,

Good job so far!
✔️ Implementation of navigation
✔️ Books can be added and removed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/navigation.js (hunk 1 above)
Comment (sumon766) [reply modules/navigation.js]: @TedLivist Thank you for your review and suggestion. I have tried using the arrow function here several times, but it doesn't work. I tried to find a solution by searching different places, but according to w3schools, we should use the arrow function if the general function does not provide the expected output. In this case, if using the arrow function, the function does not work. Here is a reference link...

https://www.w3schools.com/js/js_arrow_function.asp

File: index.js (hunk 5 above)
Comment (gemmen29) [inline index.js]: - [ ] Good job on implementing most of the changes provided by the previous reviewer but there are still `eslint-disable` line in the `index.js` page. Kindly remove this line as this defies the purpose of using linters in the first place and try to fix the error manually.
### Hint: 
I tried to remove it and I didn't find any linters errors.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @sumon766, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Highlights💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 5 above)
Comment (sumon766) [reply index.js]: @gemmen29 Thank you for the review. Actually, you are right, it does not cause any error, I just forgot to remove it :)

**Review by IngDavidTM — APPROVED** [review]
Hi @sumon766  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**